engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -64,6 +65,9 @@
     private static final DecimalFormat PERCENT_FORMAT = new DecimalFormat("0.##");
     private static final DecimalFormat SIZE_FORMAT = new DecimalFormat("#,###");
 
+    private final int verticalChunkMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+    private final int chunkHalfHeight = ChunkConstants.SIZE_Y / verticalChunkMeshSegments / 2;
```

Comment:
So, let me get this straight. 

Let's say we have a largely empty chunk, with only a 1x1 column near its center, 10 blocks tall. With a VERTICAL SEGMENTS value of 1 this column is represented by a box-shaped mesh made of 8 vertices and 12 triangles, right? While a setting of 2 would generate 12 vertices and 20 triangles, right?

Now, if I'm right, is there any reason at all to have a value that is more than 1? I can't think of one, so I'm puzzled by why this value exists in the first place.